target/linux/mediatek/image/mt7622.mk

```
@@ -240,3 +240,22 @@ define Device/ubnt_unifi-6-lr-ubootmod
   ARTIFACT/bl31-uboot.fip := bl31-uboot ubnt_unifi-6-lr
 endef
 TARGET_DEVICES += ubnt_unifi-6-lr-ubootmod
+
+define Device/xiaomi_redmi-router-ax6s
+  DEVICE_VENDOR := Xiaomi
+  DEVICE_MODEL := Redmi Router AX6S
+  DEVICE_ALT0_VENDOR := Xiaomi
+  DEVICE_ALT0_MODEL := Router AX3200
+  DEVICE_DTS := mt7622-xiaomi-redmi-router-ax6s
+  DEVICE_DTS_DIR := ../dts
```

BOARD_NAME needs to be set to "xiaomi,redmi_router_ax6s", otherwise nand_do_platform_check can't find the folder inside the tarball and it doesn't pass validation. Currently sysupgrade doesn't work because of this.

```suggestion
  BOARD_NAME := xiaomi,redmi-router-ax6s
```